setup.py

```diff
@@ -22,6 +25,8 @@
     license='BSD+',
     packages=['gnlpy'],
     package_dir={'gnlpy': '.'},
+    install_requires=requirements.txt,
```

Comment:
This will most not work. I assume it is a typo. Also, in this case, the travis.yml step should be changed and the `pip install -r requirements.txt` could be omitted.
Tests are currently failing because of this:
https://travis-ci.org/facebook/gnlpy/builds/362767334